configs/raspberrypi_config

```diff
@@ -78,25 +83,40 @@ function platform_modify() {
         run_and_log_cmd "sudo cp work/boot/${RASPBERRYPI_KERNEL_IMAGE} work/rootfs/boot/${MENDER_KERNEL_IMAGETYPE}"
     fi
 
+    # Make sure the on-board bootloader boots our bootloader (which we do via the kernel option).
+    sed -i -e '/^kernel=/d' work/boot/config.txt
+    echo "kernel=${RASPBERRYPI_BOOTLOADER_IMAGE}" >> work/boot/config.txt
+
     # Replace kernel with U-boot and add boot script
     run_and_log_cmd "sudo mkdir -p work/rootfs/uboot"
     run_and_log_cmd "sudo cp work/rpi/binaries/u-boot.bin work/boot/${RASPBERRYPI_BOOTLOADER_IMAGE}"
     run_and_log_cmd "sudo cp work/rpi/binaries/boot.scr work/boot"
     run_and_log_cmd "sudo cp work/rpi/binaries/fw_env.config work/rootfs/etc/"
     run_and_log_cmd "sudo cp work/rpi/binaries/uboot-git-log.txt work/boot"
 
+    # Disable 64-bit mode and force 32-bit mode. Even though we check for 32-bit binaries at the
+    # start of this function, some images use a 64-bit kernel even with 32-bit userspace. Due to our
+    # custom U-Boot bootloader we need to stick to 32-bit for now.
+    if ! grep -q '^arm_64bit=0' work/boot/config.txt; then
+        log_info "Forcing 32-bit kernel"
+        sed -i -e '/^arm_64bit=/d' work/boot/config.txt
+        echo "arm_64bit=0" >> work/boot/config.txt
```

Comment:
```suggestion
        sed -i -e 's/^arm_64bit=[0-9]*/arm_64bit=0/' work/boot/config.txt
```
this can be done also without the if, since we can replace only `arm_64bit=[^0]*`, but I understand that you want the log line. I am unsure about `[0-9]*` in the above, can we assume that there is always zero or more digits after `arm_64bit=`? but either way, you can do it with one sed. as most of the things. of which I know nothing, but I know you can ;-)

Reply:
Same argument as in the previous comment.